packages/ui/cypress/e2e/app/agents/read/index.spec.js

```diff
@@ -0,0 +1,69 @@
+import form from './form'
+import sourceData from './sourceData'
+import table from './table'
+
+describe('Agent read operations', () => {
+  before(() => {
+    cy.resetDevelopmentSqlDb()
+  })
+
+  form()
+  sourceData()
+  table()
+
+  describe('search', () => {
```

Comment:
I meant they should still be in their own file, /read/search.js so the pattern stays the same with importing from other files in this file